python_modules/dagster/dagster_tests/asset_defs_tests/test_assets.py

```diff
@@ -1566,12 +1566,12 @@ def load_input(self, context):
 def test_context_assets_def():
     @asset
     def a(context):
-        assert context.assets_def == a
```

Comment:
why this change?

Reply:
Normalization of an assets def changes the object identity due to the `with_attributes` call when resolving relative AssetKey deps, so checks against `AssetsDefinition` object identity need to change (not something we generally rely on, which is why it's only this test).